MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [OPTIONAL] Please it would be nice if you customize your MIT.md file so that It reflects your name and project details instead of template details. Because doing this will make your project look more professional